graphics/opencv3/Portfile

```diff
@@ -379,7 +379,7 @@ foreach python_branch {2.7} {
     }
 }
 
-set python_branches {3.5 3.6 3.7 3.8 3.9}
+set python_branches {3.5 3.6 3.7 3.8 3.9 3.10}
```

Comment:
let's remove the EOL Pythin branches `3.5` and `3.6`

Reply:
OK.
Question about the "Request feature" of Github: should I click on the "Resolve conversation" button, or do you?